include/Core/UI/Component/DashboardWidget.php

```diff
@@ -44,7 +43,12 @@ public function render() {
 			<?php _e( 'Total number of emails logged' , 'email-log' ); ?>: <strong><?php echo number_format( absint( $logs_count ), 0, ',', ',' ); ?></strong>
 		</p>
 
-		<?php Util\render_auto_delete_logs_next_run_schedule(); ?>
+		<?php
+			/**
+			 * Renders auto delete logs addon next run schedule.
```

Comment:
- Add `@since` tag
- The description of the action should say when this hook will be executed and not about where it will be used. If we decide to add details about another add-on here using the same hook, then the description that you have mentioned here doesn't hold good anymore.
- Change it to 

```
Triggered just after printing the content of the dashboard widget.

Use this hook to add custom messages to the dashboard widget.
```

Reply:
Fixed.